app/assets/javascripts/discourse/app/widgets/post-cooked.js

```diff
@@ -162,16 +161,14 @@ export default class PostCooked {
             !bestElements.has(onebox) ||
             bestElements.get(onebox) === link
           ) {
-            const title = I18n.t("topic_map.clicks", { count: lc.clicks });
-
-            link.appendChild(document.createTextNode(" "));
-            link.appendChild(
-              domFromString(
-                `<span class='badge badge-notification clicks' title='${title}'>${number(
-                  lc.clicks
-                )}</span>`
-              )[0]
-            );
+            link.setAttribute("data-clicks", number(lc.clicks));
+            const ariaLabel = `${link.textContent.trim()} ${I18n.t(
+              "post.link_clicked",
+              {
+                count: lc.clicks,
+              }
+            )}`;
+            link.setAttribute("aria-label", ariaLabel);
```

Comment:
It's neat to think about A11Y. Not sure the click count is absolutely essential, but, why not.